[DCP] Fix variable spelling

iamzainhuda opened the pull request:
Summary: tsia

Test Plan: sandcastle

Differential Revision: D57983752




cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @chauhang @d4l3k @LucasLLC

wz337 reviewed (approved):
LGTM. Thanks for fixing it!

iamzainhuda committed:
[DCP] Fix variable spelling (#127565)

Summary:
tsia


Test Plan:
sandcastle





cc mrshenli pritamdamania87 zhaojuanmao satgera gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu fegin XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 chauhang d4l3k LucasLLC

Reviewed By: wz337

Differential Revision: D57983752

Pulled By: kittenzcuddler43

iamzainhuda commented:
@pytorchbot merge

iamzainhuda commented:
@pytorchbot merge -f 'Minor changes to spelling.'
